gradle_plugin/template/gradle.properties

```diff
@@ -48,22 +48,27 @@ inspectionStrategiesStr =
 # Result folder name with tag: $(runningType)-$(tag)-$(dateTime)
 artifactTag/tag =
 # Flag to enable failing the task by throwing exception when test failed.
-enableFailTask = # Optional, Value: {true (Default), false}
+enableFailingTask = # Optional, Value: {true (Default), false}
 
 # Dynamic params, should be kept secret or may change frequently.
 # Recommend to set these variables within pipeline.
 hydraLabAPIHost = # Required, Hydra Lab host address, needs to be specified as default value is cleared for compliance.
 hydraLabAPISchema = # Optional, Hydra Lab API schema. Value: {https (Default), http}.
-appPath = # Required, path to the app file.
+appPath = # Required, path to the app file. If needReinstall = false, this value is of no use but still is required.
+appVersion = # Optional, manually passed app version. If not specified, the value will be fetched from file parser of the app by default.
 authToken = # Required, Hydra Lab user authentication token for running test.
 # Optional, the name of TEAM that the creator of authToken belongs to.
 # Value: any specified TEAM names of Hydra Lab, if not specified, will be set in Backend with creator's default TEAM name.
 teamName =
+# The name "needUninstall" is @Deprecated, use param "needReinstall" instead in the latest version.
+needReinstall/needUninstall = # Optional, flag for whether app needs to be reinstalled. Value: {true (Default), false}
```

Comment:
Could you share the reason we need to change this? Which task or deliverable is this change serves for? @olivershen-wow @zhou9584

Reply:
Reverted, and alternatively use a new param skipInstall to achieve the same goal of skipping installation.